dev-util/bruno-bin/metadata.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+    <maintainer type="person">
+        <email>anoop.md1421@gmail.com</email>
+        <name>Anoop M D</name>
+    </maintainer>
+	<maintainer type="person">
+		<email>martin-kokos@protonmail.com</email>
+		<name>Martin Mokry</name>
+	</maintainer>
+	<upstream>
+		<remote-id type="github">usebruno/bruno</remote-id>
+	</upstream>
+</pkgmetadata>
```

Comment:
Does the upstream dev use Gentoo? If not, you should put them in the <upstream> section instead.
See: https://devmanual.gentoo.org/ebuild-writing/misc-files/metadata/index.html
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
<pkgmetadata>
    <maintainer type="person">
        <email>martin-kokos@protonmail.com</email>
        <name>Martin Mokry</name>
    </maintainer>
	<upstream>
		<remote-id type="github">usebruno/bruno</remote-id>
		<maintainer type="person">
	        <email>anoop.md1421@gmail.com</email>
			<name>Anoop M D</name>
		</maintainer>
	</upstream>
</pkgmetadata>
```

Reply:
(Even if they do use Gentoo, they could also be put in `<upstream>`)

Reply:
Updated.